components/scream/data/scream_input.yaml

```diff
@@ -7,38 +7,38 @@ Atmosphere Driver:
     Schedule Type: Sequential
     Process 0:
       Process Name: Homme
-      Enable Output Fields Checks: false
-      Enable Input Fields Checks: false
+      Enable Output Fields Checks: true
```

Comment:
I apologize, I should have update this script when the syntax changed.

Would you mind changing all these entries into:
```
Enable Postcondition Checks: ...
Enable Precondition Checks: ...
```
?

Reply:
Ah, thanks for the heads up. Done.